azure-intro/java/content/lab-02/README.md

```diff
@@ -0,0 +1,227 @@
+# Deploy a Static Website
+
+In the second lab of this workshop, we're going to create a website using [static website hosting in Azure Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website).
+
+## Step 1 &mdash; Create a Resource Group
+
+We'll first create an Azure resource group, which will be used for all the other resources we're going to provision. Add the following to your imports at the top of your `App.java` file like so:
+
+```java
+import com.pulumi.azurenative.resources.ResourceGroup;
+```
+
+Next, create your Resource Group. In Pulumi, this action is done by generating a new object within the `Pulumi.run()` lambda, and assigning the resourceGroup resource to that constant, like so:
+
+```java
+var resourceGroup = new ResourceGroup("app");
+```
+
+At this stage, your complete `App.java` file should match this code:
+
+```java
+package myproject;
+
+import com.pulumi.Pulumi;
+import com.pulumi.azurenative.resources.ResourceGroup;
+
+public class App {
+    public static void main(String[] args) {
+        Pulumi.run(ctx -> {
+            var resourceGroup = new ResourceGroup("app");
+        });
+    }
+}
+```
+
+## Step 2 &mdash; Create a Storage Account
+
+Now we have the resource group created, we can now add an Azure storage account to store our HTML content for our static site.
+
+We'll need to import the storage account resource first, so add the following to your imports, near the top of your `App.java`:
+
+```java
+import com.pulumi.azurenative.storage.StorageAccount;
+import com.pulumi.azurenative.storage.StorageAccountArgs;
+import com.pulumi.azurenative.storage.enums.Kind;
+import com.pulumi.azurenative.storage.enums.SkuName;
+import com.pulumi.azurenative.storage.inputs.SkuArgs;
+```
+
+Now we can use this import to create a storage account:
+
+```java
+var storageAccount = new StorageAccount("app", StorageAccountArgs.builder()
+        .enableHttpsTrafficOnly(true)
+        .kind(Kind.StorageV2)
+        .resourceGroupName(resourceGroup.name())
+        .sku(SkuArgs.builder()
+                .name(SkuName.Standard_LRS)
+                .build())
+        .build());
+```
+
+Notice a couple of things in the code we just wrote:
+
+1. We use the name of the resource group we created (`resourceGroup.name()`, a Pulumi output) as an input to the storage account we created. This allows the Pulumi engine to know that our resource group must be created before our storage account, and that any changes to the name of the resource group name must also be applied to the storage account. For more information, see [Inputs and Outputs](https://www.pulumi.com/docs/intro/concepts/inputs-outputs/).
```

Comment:
I don't think you need to include the word engine here - it's not been explained anywhere that Pulumi has an engine?

Reply:
Fixed.